tools/clang/test/CodeGenSPIRV_Lit/intrinsics.interlocked-methods.compareexchange.reference.hlsl

```diff
@@ -2,8 +2,31 @@
 
 groupshared uint value;
 
+uint getValue() {
+  return 0;
+}
+
+uint getVector() {
```

Comment:
Should this be returning `uint2`?

Reply:
Oh right. Also noticed I hadn't commited my changed to the loop...

Reply:
Turns out those were unrelated changes... Fixed the 2 small nits and working, no change to the loop.